File: Lua/fibonacci.lua
function fib(n)
	if n <= 1 then
		return n
	else
		return fib(n-1) + fib(n-2)
	end
end

start = os.clock()

for i = 1, 50 do
	print(i.." "..fib(i))
end

print(string.format("Time taken: %.2fs\n", os.clock() - start))

File: Javascript/fibonacci.js
function fibonacci() {
	let start = performance.now();

	for (let i = 1; i <= 50; i++) {
		console.log(`${i} ${fib(i)}`);
	}
		
	console.log(`Time taken: ${(performance.now()-start)/1000}s`);
}

function fib(n) {
	if (n <= 1) {
		return n;
	} else {
		return fib(n-1) + fib(n-2);
	}
}
